cli/src/main/java/com/nextdoor/bender/CreateSchema.java

```diff
@@ -4,13 +4,12 @@
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- *
- * Copyright 2017 Nextdoor.com, Inc
+ * Unless required by applicable law or agreed to in writing, software distributed under the
```

Comment:
The line lengths changed a little bit because we were right over the 100char line char limit in the google style guide (https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml) (by 2 chars)..